test/wilson/dom_test.cljs

```diff
@@ -33,16 +33,13 @@
     (is (= (d/table [:a-key :some-key :some-other-key]
                     [{:a-key (d/label "warning" "h")
                       :some-key "i"
-                      :some-other-key "j"
-                      :hidden "hidden"}
```

Comment:
You shouldn't get rid of these `:hidden` row entries; they test that the produced table only has the specified keys.